assets/js/components/channels/ProviderModal.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from "react"
+import { translate } from "react-i18next"
+import { ConfirmationModal } from "../ui"
+
+export const ProviderModal = ({
+  t,
+  provider,
+  index,
+  friendlyName,
+  multiple,
+  onConfirm,
+  loading,
+  surveys,
+}) => {
+  let name = `${provider[0].toUpperCase()}${provider.slice(1)}`
+  if (multiple) name = `${name} (${friendlyName})`
+
+  return (
+    <ConfirmationModal
+      modalId={`${provider}Modal-${index}`}
+      header={t("Turn off {{name}}", { name })}
+      confirmationText={t("Yes")}
+      onConfirm={onConfirm}
+      style={{ maxWidth: "600px" }}
+      showCancel
+    >
+      <div>
+        <p>{t("Do you want to delete the channels provided by {{name}}?", { name })}</p>
+
+        <div className="provider-surveys">
+          {loading ? <span>{t("Searching active surveys...")}</span> :
+            surveys.length == 0 ? <span>{t("No active surveys")}</span> :
+              <div>
+                <span>{t("Active surveys")}</span>
```

Comment:
```suggestion
                <span>{t("These surveys are active, using channels from this provider. Deleting the channels will interrupt the surveys.")}</span>
```

May be a better wording, but I'd try to explain a bit more why this is relevant to the user.